### ISSUE TITLE
Creating a graph should return metadata back

### ISSUE THREAD

**Issue by QubitPi** [paion-data/aristotle#17]
> [!IMPORTANT]  
> 
> @Doom9527 Please make sure local API doc server is running at http://localhost:8080/doc.html first to be able to click on some of the links below

[It is a standard practice to return the persisted entity ID after a POST operation](https://jsonapi.org/format/#crud-creating-responses-201). Creating a graph, however, [returns a plain empty data](http://localhost:8080/doc.html#/default/Node%20controller%20for%20handling%20graph%20node-related%20operations/createGraphAndBindGraphAndNodeUsingPOST):

![page 3](https://github.com/user-attachments/assets/198321e8-da81-4bb2-a742-7a1e11460fc4)

Although _we_ know graph ID should be retrieved by a [separate endpoint](http://localhost:8080/doc.html#/default/User%20controller%20for%20handling%20user-related%20operations/getGraphByUserUidcidUsingGET) and I think you did what's been asked to do, the outsiders will have a different experience. __Our outside user will assume Aristotle is a JSON API webservice, client user thus would assume the standard is that POSTing a graph will return its newly created ID so that client user can use that ID to query the graph later. Complying the [standard](https://jsonapi.org/format/) is very beneficiary to your career, because it makes a difference between a harsh frontend colleague who doesn't know what you are doing and a very collaborative colleague who always knows your (standard) assumption and work with you happily__

Another reason to return the graph meta in the same POST request is to allow frontend to _save network I/O_ by avoiding that [extra graph ID list request](http://localhost:8080/doc.html#/default/User%20controller%20for%20handling%20user-related%20operations/getGraphByUserUidcidUsingGET) whenever possible

In conclusion, we should return the entire graph object that includes not just ID, but also nodes and links (with their titles, descs, ID, etc) as well
